jsonrpc/txpool_endpoint.go

```diff
@@ -0,0 +1,153 @@
+package jsonrpc
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/0xPolygon/minimal/types"
+)
+
+// Txpool is the txpool jsonrpc endpoint
+type Txpool struct {
+  d *Dispatcher
+}
+
+type ContentResponse struct {
+  Pending   map[types.Address]map[uint64]*gethTransaction   `json:"pending"`
+  Queued    map[types.Address]map[uint64]*gethTransaction   `json:"queued"`
+}
+
+type InspectResponse struct {
+  Pending   map[string]map[string]string  `json:"pending"`
+  Queued    map[string]map[string]string  `json:"queued"`
+}
+
+type StatusResponse struct {
+  Pending   uint64  `json:"pending"`
+  Queued    uint64  `json:"queued"`
+}
+
+type gethTransaction struct {
+  Nonce       argUint64      `json:"nonce"`
+  GasPrice    argBig         `json:"gasPrice"`
+  Gas         argUint64      `json:"gas"`
+  To          *types.Address `json:"to"`
+  Value       argBig         `json:"value"`
+  Input       argBytes       `json:"input"`
+  V           argByte        `json:"v"`
+  R           argBytes       `json:"r"`
+  S           argBytes       `json:"s"`
+  Hash        types.Hash     `json:"hash"`
+  From        types.Address  `json:"from"`
+  BlockHash   types.Hash     `json:"blockHash"`
+  BlockNumber interface{}    `json:"blockNumber"`
+  TxIndex     interface{}    `json:"transactionIndex"`
+}
+
+func toGethTransaction(t *types.Transaction) *gethTransaction {
+  if t.R == nil {
+    t.R = []byte{0}
+  }
+  if t.S == nil {
+    t.S = []byte{0}
+  }
+
+  return &gethTransaction{
+    Nonce:        argUint64(t.Nonce),
+    GasPrice:     argBig(*t.GasPrice),
+    Gas:          argUint64(t.Gas),
+    To:           t.To,
+    Value:        argBig(*t.Value),
+    Input:        argBytes(t.Input),
+    V:            argByte(t.V),
+    R:            argBytes(t.R),
+    S:            argBytes(t.S),
+    Hash:         t.Hash,
+    From:         t.From,
+    BlockHash:    types.ZeroHash,
+    BlockNumber:  nil,
+    TxIndex:      nil,
+  }
+}
+
+/** 
+ * Content - implemented according to https://geth.ethereum.org/docs/rpc/ns-txpool#txpool_content
+ */
```

Comment:
This is probably a nitpick, but throughout the entire project we've never used comments like this. Please place multiline comments in // instead of /* */. Function comments should have a short description as to what they do, and you can additionaly link to the spec after the explanation